docs/modules/hdfs/pages/usage-guide/operations/graceful-shutdown.adoc

```diff
@@ -1,6 +1,33 @@
 = Graceful shutdown
 
-Graceful shutdown of HDFS nodes is either not supported by the product itself
-or we have not implemented it yet.
+You can configure the graceful shutdown as described in xref:concepts:operations/pod_placement.adoc[].
 
-Outstanding implementation work for the graceful shutdowns of all products where this functionality is relevant is tracked in https://github.com/stackabletech/issues/issues/357
+== JournalNodes
+
+As a default, JournalNodes have `15` minutes to terminate gracefully.
```

Comment:
Nit: the value is "15 minutes", they should (IMO) be formatted together.

Reply:
Also: where does this number come from? Do we have a reason to expect it to take this long?

Reply:
To be honest it's a time period we did came up and considered to be a worst case, while being more on the pessimistic side

Reply:
Can this be resolved? :)